S2VX.Game/SongSelection/SongSelectionScreen.cs

```diff
@@ -0,0 +1,37 @@
+﻿using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Screens;
+using osuTK;
+using S2VX.Game.Editor;
+
+namespace S2VX.Game.Play {
+    public class SongSelectionScreen : Screen {
+        private static Vector2 InputSize { get; } = new Vector2(100, 30);
```

Comment:
General question: is there any particular reason to use `readonly` over `const` in cases where we're just setting the value at the start and never changing it ever again?

Reply:
Oh right, this is calling a constructor and so `const` won't work because it's not evaluated at compile-time. This can be set to `readonly` though.

Reply:
I don't think you can use `readonly` with properties. 
![image](https://user-images.githubusercontent.com/3955187/95827360-7c29eb00-0ce8-11eb-87d2-874ce4765cd9.png)

I've essentially done the same thing with properties by only allowing a getter.